ygen/enumgen.go

```diff
@@ -224,11 +224,23 @@ func (s *enumSet) identityBaseKey(i *yang.Identity) string {
 	return fmt.Sprintf("%s/%s", definingModYANGName, i.Name)
 }
 
+// nodePath returns the full path of the node from the module name.
+func nodePath(n yang.Node) string {
```

Comment:
It was interesting to me that we didn't have a `util` function for this. We could consider putting it there if we think that this will be generally useful to others. I'm fine with it staying here though.